src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1211,11 +1211,22 @@ dataset.exportBtn.itemLabel.oai_ore=OAI_ORE
 dataset.exportBtn.itemLabel.dataciteOpenAIRE=OpenAIRE
 metrics.title=Metrics
 metrics.title.tip=View more metrics information
-metrics.comingsoon=Coming soon...
-metrics.views=Views
+metrics.dataset.title=Dataset Metrics
+metrics.dataset.tip.default=Aggregated metrics for this dataset.
+metrics.dataset.tip.makedatacount=Metrics collected using Make Data Count standards.
+metrics.dataset.views.tip=Dataset views are combined with aggregated file views.
+metrics.dataset.downloads.default.tip=Total aggregated downloads of files in this dataset.
+metrics.dataset.downloads.makedatacount.tip=Each file download (single or multiple files) increments the download count.
+metrics.dataset.citations.tip=Click for a list of citation URLs.
+metrics.file.title=File Metrics
+metrics.file.tip.default=Metrics for this individual file.
+metrics.file.tip.makedatacount=Individual file downloads are tracked but this is not part of the Make Data Count standard.
+metrics.file.downloads.tip=Total downloads of this file.
+metrics.views={0, choice, 0#Views|1#View|2#Views}
 metrics.downloads={0, choice, 0#Downloads|1#Download|2#Downloads}
-metrics.citations=Citations
-metrics.shares=Shares
+metrics.citations={0, choice, 0#Citations|1#Citation|2#Citations}
+metrics.citations.dialog.header=Dataset Citations
+metrics.citations.dialog.help=Citations for this dataset are retrieved from <a href="https://www.crossref.org" title="Crossref" target="_blank">Crossref</a> via DataCite using Make Data Count standards. For more information about dataset metrics, please refer to the <a href="{0}/{1}/user/dataset-management.html#dataset-metrics-user" title="Dataset Metrics - Dataverse User Guide" target="_blank">User Guide</a>.
```

Comment:
@pdurbin I would recommend that the UI is not the appropriate place for an external link to find more information, but instead put this link in the guides, which we already link to in the next sentence.

Reply:
@mheppler thanks (especially for adding the links to the guides!). @djbrooke is going to put the finishing touches on this. (There's a new "to do" column in the spreadsheet.)

As of my commit, here's how the popup looks with two links:

<img width="775" alt="Screen Shot 2019-07-22 at 3 27 32 PM" src="https://user-images.githubusercontent.com/21006/61659430-6bc9c280-ac95-11e9-9c27-9d04a5a9ef53.png">

I believe we're going for slightly shorter wording, like this:

<img width="463" alt="Screen Shot 2019-07-22 at 3 27 46 PM" src="https://user-images.githubusercontent.com/21006/61659429-6bc9c280-ac95-11e9-8a67-bb942582e281.png">

@TaniaSchlatter will be reviewing all this. (I'm continuing to build and push the latest to http://ec2-3-86-35-164.compute-1.amazonaws.com/